operate/client/e2e-playwright/tests/processInstanceVariables.spec.ts

```diff
@@ -81,8 +81,12 @@ test.describe('Process Instance Variables', () => {
     await expect(page.getByText('"editedTestValue"')).toBeVisible();
   });
 
-  test('Add variables', async ({page, processInstancePage, processesPage}) => {
-    test.setTimeout(DEFAULT_TEST_TIMEOUT + 1 * 15000); // 15 seconds for each applied operation in this test
```

Comment:
I removed this in favor of `test.slow()`.

Before: 45s timeout (30s + 15s)
After: 90s timeout (30s * 3)